src/Uno.UWP/Media/Capture/CameraCaptureUI.other.cs

```diff
@@ -8,7 +8,7 @@ namespace Windows.Media.Capture
 {
 	public partial class CameraCaptureUI
 	{
-		private async Task<StorageFile> CaptureFile(CancellationToken arg, CameraCaptureUIMode mode)
+		private Task<StorageFile> CaptureFile(CancellationToken arg, CameraCaptureUIMode mode)
```

Comment:
This is not semantically the same. We want to return a null StorageFile, not a null Task.